src/main/scala/net/bdew/ae2stuff/machines/wireless/TileWireless.scala

```diff
@@ -71,27 +73,48 @@ class TileWireless
   })
 
   def doLink(other: TileWireless): Boolean = {
-    if (other.link.isEmpty) {
+    if (other.link.isEmpty && !(isHub || other.isHub)) {
```

Comment:
I think `&& !isHub && !other.isHub` would be clearer here

Reply:
done